apps/web/app/services/client/api/task-sizes.ts

```diff
@@ -21,6 +21,9 @@ export function deleteTaskSizesAPI(id: string) {
 	return api.delete<DeleteResponse>(`/task-sizes/${id}`);
 }
 
-export function getTaskSizesList(tenantId: string, organizationId: string, activeTeamId: string | null) {
-	return api.get(`/task-sizes?tenantId=${tenantId}&organizationId=${organizationId}&activeTeamId=${activeTeamId}`);
+export async function getTaskSizesList(tenantId: string, organizationId: string, activeTeamId: string | null) {
+	const endpoint = `/task-sizes?tenantId=${tenantId}&organizationId=${organizationId}&organizationTeamId=${activeTeamId}`;
+
+	// API call via Proxy Nextjs /api routes
+	return process.env.NEXT_PUBLIC_GAUZY_API_SERVER_URL ? apiDirect.get(endpoint) : api.get(endpoint);
```

Comment:
@badalkhatri0924 still, I don't get why we can't encapsulate that check for `process.env.NEXT_PUBLIC_GAUZY_API_SERVER_URL` inside some api.get() method?? 

get(endpoint, isDirect) { 
  return isDirect && process.env.NEXT_PUBLIC_GAUZY_API_SERVER_URL ? apiDirect.get(endpoint) : api.get(endpoint);
}

and so in the method above just do return api.get(endpoint, true); when we know that method works with direct and do `api.get(endpoint)` as before when we know when method don't work yet with direct???

Reply:
@evereq please check again, I've updated code.